server/embed/config_test.go

```diff
@@ -289,3 +289,64 @@ func TestPeerURLsMapAndTokenFromSRV(t *testing.T) {
 		}
 	}
 }
+
+func TestLogRotate(t *testing.T) {
+	tests := []struct {
+		name              string
+		logOutputs        []string
+		logRotationConfig string
+		wantErr           bool
+		wantErrMsg        error
+	}{
+		{
+			name:              "mixed log output targets",
+			logOutputs:        []string{"stderr", "/tmp/path"},
+			logRotationConfig: `{"maxsize": 1}`,
```

Comment:
What's the behavior of simplest default config: 

```
 --logRotationConfig='{}'
```
? Maybe worth testing / documenting as part of flag description.